src/main/service/tableService.ts

```diff
@@ -0,0 +1,51 @@
+import moment from 'moment';
+
+export class TableService {
```

Comment:
i was thinking this would be more generic, but if we cant, then lets change it back from TableService to SubscriptionSerivce, 
we cant use this service to create tables for anything else as is so maybe just rename